app/views/home/businesses.html.erb

```diff
@@ -21,49 +21,49 @@
 </div>
 
 <div class="container-lg py-6 py-3 px-3 text-center">
-  <h2 class="alt-h2">Your business depends on it</h2>
+  <h2 class="alt-h2"><%=t "Your business depends on it" %></h2>
 
   <p class="alt-lead text-gray mx-auto text-center mb-6">
-    Companies need open source software to run effectively.
+  <%=t "Companies need open source software to run effectively" %>
```

Comment:
`.` may be recognized special character for rails i18n system.
e.g.   `foo.boo.key`  may look for 
```
-foo
  -boo
     key = value
```

To avoid interference with this feature, I'd like not to use `.` in I18n key name in this way.